src/solvers/truncated_conjugate_gradient_descent.jl

```diff
@@ -121,12 +132,21 @@ function truncated_conjugate_gradient_descent!(
         StopWhenCurvatureIsNegative(),
         StopWhenModelIncreased(),
     ),
+    project_to_tangent!::Proj=copyto!,
     return_options=false,
     kwargs..., #collect rest
-) where {TF,TG,TH,Tprec}
+) where {TF,TG,TH,Tprec,Proj}
     p = HessianProblem(M, F, gradF, H, preconditioner; evaluation=evaluation)
     o = TruncatedConjugateGradientOptions(
-        p, x, η, trust_region_radius, randomize; θ=θ, κ=κ, stop=stopping_criterion
+        p,
+        x,
+        η,
+        trust_region_radius,
+        randomize;
+        θ = θ,
+        κ = κ,
+        stop = stopping_criterion,
+        (project_to_tangent!)=project_to_tangent!,
```

Comment:
Hm, I just noticed 
```suggestion
        project_to_tangent! = project_to_tangent!,
```
I think this should work as well and looks nicer. Or was this a format by the formatter?